react/src/reactTextMask.js

```diff
@@ -66,7 +51,24 @@ export const MaskedInput = React.createClass({
       this.props.onChange(event)
     }
   }
-})
+}
+
+MaskedInput.propTypes = {
```

Comment:
why move this code block to the bottom? can this be defined on the class like it was? ..can it be put back where it was? or is there a valid reason for moving it